source/Magritte-Model.package/MAMemento.class/instance/doesNotUnderstand..st

```diff
@@ -0,0 +1,4 @@
+reflective operations
```

Comment:
I'm not a big fan of DNU magic but if you find that a life saver then be it :) Two things. If this introduces doesNotUnderstand: at the level of MAMemento it would be good to implement readUsing: in the same class with sublassResponsibility. And why did you only do reading and not writing?

Reply:
- Regarding the suspicion of DNU magic, I'm in the same boat, but I finally cracked after repeatedly seeing code like this:
```smalltalk
MyObject>>#descriptionContainer
	^ super descriptionContainer
		addCondition: [ :obj | (obj readUsing: includesDay) or: [ obj readUsing: includesNight ] ];
		addCondition: [ :obj | ((obj readUsing: payCodes) size between: 1 and: 2) and: [ (obj readUsing: payCodes) allSatisfy: [ :code | (obj readUsing: payFactorsFromHours) includes: code factor ] ] ];
		addCondition: [ :obj | (obj readUsing: payRate) <= (obj readUsing: maxPayRate);
		yourself
``` 
which can now be simplified to the slightly less depressing:
```smalltalk
MyObject>>#descriptionContainer
	^ super descriptionContainer
		addCondition: [ :obj | obj includesDay or: [ obj includesNight ] ];
		addCondition: [ :obj | (self payCodes size between: 1 and: 2) and: [ self payCodes allSatisfy: [ :code | self payFactorsFromHours includes: code factor ] ] ];
		addCondition: [ :obj | obj payRate <= obj maxPayRate;
		yourself
``` 
which at least saves quite a few parentheses.
		
- Re why only reading, it's because the use case is [here](
https://github.com/seandenigris/Magritte3/wiki/Conditions#field-relationship), so I did the minimum to make that work
- Re `#readUsing:`, see commit below